src/dvu/gfx/visualization.ts

```diff
@@ -19,12 +19,12 @@ export class CompositeVisualization extends PictureCommand {
     super(this.name, this.getPictureCommandInterface())
   }
 
-  execute(context: PictureContext, scope: Scope = new Scope()): Picture {
+  execute(scope: Scope = new Scope()): Picture {
```

Comment:
Squash these 2 commits into one, we can merge it but we'll still need to think about this again